packages/manager/modules/pci/src/projects/project/storages/containers/container/object/add/add.controller.js

```diff
@@ -1,17 +1,32 @@
 import get from 'lodash/get';
+import {
+  STORAGE_PRICES_LINK,
+  OBJECT_CONTAINER_STORAGE_CLASS,
+} from '../../../containers.constants';
 
 export default class PciBlockStorageContainersContainerObjectAddController {
   /* @ngInject */
-  constructor($translate, atInternet, PciProjectStorageContainersService) {
+  constructor(
+    $translate,
+    atInternet,
+    coreConfig,
+    PciProjectStorageContainersService,
+  ) {
     this.$translate = $translate;
     this.atInternet = atInternet;
+    this.coreConfig = coreConfig;
+    this.STORAGE_PRICES_LINK = STORAGE_PRICES_LINK;
```

Comment:
It seems that this is not used in the template, therefore you can directly use STORAGE_PRICES_LINK in the controller 
```suggestion
```